exercises/E-strings-concatenation/exercise.js

```diff
@@ -1,3 +1,10 @@
 // Start by creating a variable `message`
 
+let message="Hello, my name is "
+let myName="Abdulaziz"
+
+let greeting=message+myName;
+message +=myName;
+
 console.log(message);
```

Comment:
Hey Abdulaziz, this looks good, I see you're experimenting both ways to print a message, this is another method:

            var message="Hello, my name is ";
            var name="Abdulaziz";
            console.log(message+name);

You can do the string concatenation inside the console log to achieve the same thing, but needless to say, well done 😁.